src/Microsoft.OData.Core/Json/ODataUtf8JsonWriter.cs

```diff
@@ -13,17 +13,19 @@
 using System.Text.Json;
 using System.Text.Encodings.Web;
 using Microsoft.OData.Edm;
+using System.Threading.Tasks;
 
 namespace Microsoft.OData.Json
 {
     /// <summary>
-    /// Implementation of <see cref="IJsonWriter"/> that is based on
+    /// Implementation of <see cref="IJsonWriter"/> and <see cref="IJsonWriterAsync"/>that is based on

```

Comment:
nit: is this statement complete?

Reply:
Yes it's complete. The statement continues on the next line, i.e.:
```
... is based on
<see cref="Utf8JsonWriter">.
```